scripts/build.ps1

```diff
@@ -152,6 +152,8 @@ if ("" -eq $Platform) {
     }
 }
 
+$env:CLOG_DEVELOPMENT_MODE = 1
```

Comment:
This is not supposed to be checked in.

Reply:
I know, but it fails in CI and I just want to see if it builds at all.

Reply:
If it does build, it means we need to rebuild locally, to update the sidecar, and then the CI won't need it.

Reply:
One flaw in this I noticed is runs on windows don't seem to affect linux. So if we ever updated, we'd have to run manually on all platforms. Not too difficult until we add mac.

Reply:
Perhaps a special pipeline that runs all platforms up update the config for us? Or have that automatically happen for PR CI runs?